iceoryx_hoofs/include/iceoryx_hoofs/cxx/type_traits.hpp

```diff
@@ -75,11 +67,13 @@ struct is_invocable
     // This variant is chosen when Callable(ArgTypes) successfully resolves to a valid type, i.e. is invocable.
     /// @note result_of is deprecated, switch to invoke_result in C++17
     template <typename C, typename... As>
-    static constexpr std::true_type test(typename cxx::invoke_result<C, As...>::type* f IOX_MAYBE_UNUSED) noexcept
+    static constexpr std::true_type test(typename platform::invoke_result<C, As...>::type* f IOX_MAYBE_UNUSED) noexcept
     {
         return {};
     }
 
+    // AXIVION Next Construct AutosarC++19_03-A8.4.1 : we require a SFINEA failure case where all
+    // parameter types (non invokable ones) are allowed, this can be achieved with variadic arguments
     // This is chosen if Callable(ArgTypes) does not resolve to a valid type.
     template <typename C, typename... As>
     /// @NOLINTJUSTIFICATION we require a SFINEA failure case where all parameter types (non invokable ones) are
```

Comment:
You can remove this and the following line. It's enough if the justification is in the Axivion line.

Reply:
@mossmaurice Sure